media/css/mozorg/webvision.scss

```diff
@@ -70,37 +70,55 @@ $image-path: '/media/protocol/img';
 
     @media #{$mq-md} {
         flex-direction: row;
-
-        .c-webvision-article {
-            margin-left: $layout-sm;
-        }
     }
 
-    .webvision-sidebar {
-        position: sticky;
-        max-height: 100vh;
-        top: 0;
-        overflow-y: auto;
-        overflow-x: hidden;
-    }
-
-    .mzp-c-sidemenu-main {
+    .c-webvision-article {
 
         @media #{$mq-md} {
-            padding: $layout-lg 0;
+            margin-right: 0;
         }
 
-        @media #{$mq-xl} {
-            width: 350px;
+        @media #{$mq-lg} {
+            margin-right: $layout-2xl;
         }
     }
+}
+
+.webvision-sidebar {
+    position: sticky;
+    max-height: 100vh;
+    top: 80px;
+    overflow-y: auto;
+    overflow-x: hidden;
+}
+
+.mzp-c-sidemenu-main {
+
+    @media #{$mq-md} {
+        padding-bottom: $layout-xl;
+    }
+
+    @media #{$mq-xl} {
+        width: 350px;
+    }
+}
+
+.toc {
+    padding: $spacing-xs;
 
-    .toc {
-        padding: $spacing-xs;
+    ul {
+        li {
+            margin: $spacing-sm;
```

Comment:
You can make this `margin-left`, we don't need margins on any other side and this can be just for indentation.